Update Cisco

shengbo-ma opened the pull request:
<!-- 
INSTRUCTIONS

Thank you for contributing to new grad positions! Please follow this format when submitting a PR:
* Title should be capitalized i.e. ("Add Microsoft" not "add Microsoft")
* Do not end the subject line with a period
* Use the imperative mood ("Add Microsoft" not "Adds Microsoft")
* When adding position use "Add" ("Add Microsoft")
* When removing a position use ("Close Microsoft") and mark that position with a strikethrough
-->

qc542 commented:
![Screenshot_20220917_092011](https://user-images.githubusercontent.com/36024574/190859064-f51de265-b9c5-40a2-9e54-d14651c8e96a.jpg)

shengbo-ma commented:
Fixed

> ![Screenshot_20220917_092011](https://user-images.githubusercontent.com/36024574/190859064-f51de265-b9c5-40a2-9e54-d14651c8e96a.jpg)

Fixed

thinkfurther commented:
no visa support
